### PR TITLE
Allow short relative paths in imports

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR fixes a crash in `FileReference.directory()`. Currently, the method throws an exception if the file reference has no directory, i.e. it's a simple file name (possibly, with the "`./`" prefix). This PR fixes that crash.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: tools/tool-base/src/test/java/io/spine/code/fs/js/FileReferenceTest.java [hunk 1]
```diff
@@ -63,4 +64,13 @@ void directorySkipRelative() {
         assertEquals("foo/bar", fileReference.directory()
                                              .value());
     }
+
+    @Test
+    @DisplayName("obtain the empty directory path")
+    void emptyDirectory() {
+        FileReference file = FileReference.of("./neighbour.txt");
```
Comment (armiol) [inline tools/tool-base/src/test/java/io/spine/code/fs/js/FileReferenceTest.java:71]: Honestly, I would expect that the `/` symbol here would be a system-specific path delimiter. I don't know, but it seems to me that some Windows versions would gladly interpret `.\neighbour.txt` as a file in the current folder.

File: tools/tool-base/src/main/java/io/spine/code/fs/DirectoryReference.java [hunk 2]
```diff
@@ -54,10 +54,17 @@ public static DirectoryReference of(String value) {
         return new DirectoryReference(value);
     }
 
+    /**
+     * Obtains the reference to the current directory.
+     */
+    public static DirectoryReference currentDir() {
```
Comment (armiol) [inline tools/tool-base/src/main/java/io/spine/code/fs/DirectoryReference.java:60]: Similar to my comment below, I would expect that the reference to the current dir would be either `"."` or `"." + File.separatorChar`.

**Review by armiol — COMMENTED** [review]
@dmdashenkov please see my comments.

File: tools/tool-base/src/test/java/io/spine/code/fs/js/FileReferenceTest.java (hunk 1 above)
Comment (dmdashenkov) [reply tools/tool-base/src/test/java/io/spine/code/fs/js/FileReferenceTest.java:71]: As this is an *imported* file reference, we only use this `/` type of slashes.

**Review by armiol — APPROVED** [review]

**Pull request merged**